R/print.data.table.R

```diff
@@ -101,8 +110,23 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
       IDate = "<IDat>", integer64 = "<i64>", raw = "<raw>",
       expression = "<expr>", ordered = "<ord>")
     classes = classes1(x)
+    col_names <- colnames(toprint)
+    classes <- sapply(col_names, function(col_name) {
```

Comment:
Instead of overwriting `classes` by re-creating it anew, can you append the index markers to the variable `classes` created two lines above?